app/models/session_activation.rb

```diff
@@ -53,4 +59,22 @@ def exclusive(id)
       where('session_id != ?', id).destroy_all
     end
   end
+
+  private
+
+  def assign_user_agent
+    self.user_agent = '' if user_agent.nil?
+  end
+
+  def assign_access_token
+    superapp = Doorkeeper::Application.find_by(superapp: true)
+
+    return if superapp.nil?
```

Comment:
Looks like our `db:migrate` does also `db:setup` now, but it was introduced at v1.4rc4...
https://github.com/tootsuite/mastodon/commit/256e3adc1d9508423aab8fcfb13745c9f85ff948#diff-1bf337030f4e56fa36281d23cf9e2082